tools/flowc/backends/nim/natives/net/url_parameter/getAllUrlParametersArray.nim

```diff
@@ -1,2 +1,11 @@
+import os
+
 proc getAllUrlParametersArray*(): seq[seq[string]] =
-  return newSeq[seq[string]]()
\ No newline at end of file
+  var params = filter(commandLineParams(), proc(param: string): bool = param.len > 0)
+  return map(params, proc(param: string): seq[string] =
+    let p = split(param, "=")
+    if p.len == 1:
```

Comment:
1) params:
a==k
a=k=p
=k=p

2) filtermap instead of filter + map
or 3
3) iterator split

Reply:
`proc getAllUrlParametersArray*(): seq[seq[string]] =
  let params = filter(commandLineParams(), proc(p: string): bool = p.len > 0)
  return map(params, proc(p: string): seq[string] =
    let eq_ind = find(p, "=")
    if eq_ind == -1:
      return @[p, ""]
    else:
      return @[p[0 .. (eq_ind - 1)], p[(eq_ind + 1) .. (p.len - 1)]]
  )`